packages/pwa/app/components/_app-config/index.jsx

```diff
@@ -40,7 +41,7 @@ const getLocale = (locals) => {
     const localeUrl = originalUrl && originalUrl.split('/')[1]
```

Comment:
This code is not solid. If you goto the url `http://localhost:3000/it-IT?site=broken` you'd expect to see the site in italian, but you see it in the default locale because of this split isn't really the smartest code. You might have to first remove the query string if there is one.